packages/account/src/Components/poi/status/upload-complete/__tests__/upload-complete.spec.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Button } from '@deriv/components';
+import { PlatformContext } from '@deriv/shared';
+import { UploadComplete } from '../upload-complete.jsx';
+
+jest.mock('Components/poa/poa-button', () => jest.fn(() => <div data-testid='poa-button' />));
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Icon: jest.fn(() => <div data-testid='dt_mocked_icon' />),
+    };
+});
+
+describe('<UploadComplete />', () => {
+    const successful_upload_message = /Your proof of identity was submitted successfully/i;
+    const poi_under_review_message = /We’ll review your document and notify you of its status within 1 to 2 hours./i;
```

Comment:
I think we can use lowercase here and further in the code in regexp because we have here '/i' param, with allows the string to insensitive for big/little letters